.github/workflows/ci.yml

```diff
@@ -124,6 +124,11 @@ jobs:
         #
         # Ref: https://github.com/tox-dev/tox/issues/3193
         run: tox -vv --notest -p auto --parallel-live
+      - name: Set PYTEST_ADDOPTS for py38
+        if: matrix.python-version == '3.8'
+        shell: bash
+        run: |
+          echo "PYTEST_ADDOPTS=--cov-fail-under=98" >> $GITHUB_ENV
```

Comment:
I have all sorts of thoughts about improvements (including ones we're working on!), but I'm okay with this as a current solution that gets us unblocked.

@webknjaz, I want to confirm that you're also alright with this solution before we merge. For that reason I'm not yet adding this to the merge-queue.

Reply:
Yeah, I actually said that it's fine but that thread go collapsed and isn't as visible.

FWIW, other people can't reset blocking reviews in PRs so I'll go ahead and change it to approved.